src/icons/register-planet.svg

```diff
@@ -0,0 +1,30 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 1362.1 2280.2">
+    <style type="text/css">
+        svg{opacity:.3}
+        .st1,.st3{opacity:.7;fill:#f2f2f2}.st3{opacity:.5}
```

Comment:
Be careful, because this creates global class names that can affect other svgs with the same classes. It's probably fine in this case, but worth a note.